src/main/java/org/terasology/blockdetector/systems/BlockDetectorSystemImpl.java

```diff
@@ -67,6 +63,10 @@
     @In
     private LocalPlayer localPlayer;
 
+    public void setLocalPlayer(LocalPlayer set){
```

Comment:
move these functions below the declaration.

Reply:
Agreed

Reply:
There has to a space after the parentheses.

`set){` -> `set) {`